src/actions/noting/create-note-from-selection.js

```diff
@@ -0,0 +1,94 @@
+var _ = require('lodash');
+var VText = require('vtree/vtext');
+
+var isVFocus = require('../../utils/vfocus/is-vfocus');
+var errorHandle = require('../../utils/error-handle');
+var findTextBetweenScribeMarkers = require('../../utils/noting/find-text-between-scribe-markers');
+var getNoteDataAttributes = require('../../utils/get-note-data-attrs');
+var wrapInNote = require('./wrap-in-note');
+var removeErroniousBrTags = require('./remove-erroneous-br-tags');
+var removeScribeMarkers = require('./remove-scribe-markers');
+var findLastNoteSegment = require('../../utils/noting/find-last-note-segment');
+var findEntireNote = require('../../utils/noting/find-entire-note');
+var resetnoteSegmentClasses = require('./reset-note-segment-classes');
+var notesCache = require('../../utils/noting/note-cache');
+var isNotWithinNote = require('../../utils/noting/is-not-within-note');
+var isParagraph = require('../../utils/vfocus/is-paragraph');
+var createVirtualScribeMarker = require('../../utils/create-virtual-scribe-marker');
+var isNotEmpty = require('../../utils/vfocus/is-not-empty');
+var removeEmptyNotes = require('../../actions/noting/remove-empty-notes');
+
+// treeFocus: tree focus of tree containing two scribe markers
+// Note that we will mutate the tree.
+module.exports = function createNoteFromSelection(focus){
+
+  if (!isVFocus(focus)){
```

Comment:
As mentioned previously theres a `.jscsrc` on the way which will fix these if you bear it in the code base for now?